<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chip_options</class>
 <widget class="QDialog" name="chip_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_shape_round">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Round</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_shape_square">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_shape_triangle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_shape_xo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X/O</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>chip_options</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>chip_options</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
